README.md

```diff
@@ -20,13 +20,12 @@
 # Table of Contents
 
 -   [Introduction](#introduction)
--   [Screenshots](#screenshots)
+-   [Installation & Usage](#installation--usage)
+    - [Recommended setup for `lazy.nvim`](#recommended-setup-for-lazynvim)
 -   [Requirements](#requirements)
--   [Installation](#installation)
--   [Setup](#setup)
-    -   [Extensions](#extensions)
 -   [Commands](#commands)
 -   [Registries](#registries)
+-   [Screenshots](#screenshots)
 -   [Configuration](#configuration)
 
 # Introduction
```

Comment:
Just a nit since we are already reworking the docs: In Markdown, there is the convention to only use one h1 per file: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md001---heading-levels-should-only-increment-by-one-level-at-a-time